sdk/src/cameras/ad-96tof1-ebz/calibration_96tof1.cpp

```diff
@@ -113,8 +114,7 @@ aditof::Status Calibration96Tof1::displayCalMap() const {
 SaveCalMap - Saves the entire calibration map as binary to a file.
 \device - Pointer to a device instance
```

Comment:
Should be updated in order to match the new implementation (ex: `\eeprom - Reference to an eeprom instance`)

Reply:
fixed